src/tools/wpa/DataModel/QuicObjectSet.cs

```diff
@@ -77,9 +75,9 @@ public override int GetHashCode()
 
     public sealed class QuicObjectSet<T> where T : class, IQuicObject
     {
-        private Dictionary<QuicObjectKey, T> activeTable = new Dictionary<QuicObjectKey, T>();
+        internal Dictionary<QuicObjectKey, T> activeTable = new Dictionary<QuicObjectKey, T>();
```

Comment:
If these are going to be internal but not properties, make them `readonly`

Reply:
Realistically there are some other fields that could use readonly as well in other places, but I'm fine doing that in a later cleanup.